pkg/bot/slack_renderer.go

```diff
@@ -253,30 +258,32 @@ func (b *SlackRenderer) renderContext(in []api.ContextItem) []slack.Block {
 //
 //  2. Without description: all in the same row. For example:
 //     [Button "Get Pods"] [Button "Get Deployments"]
-func (b *SlackRenderer) renderButtons(in api.Buttons) []slack.Block {
+func (b *SlackRenderer) renderButtons(in api.Buttons, selects *slack.ActionBlock) ([]slack.Block, bool) {
 	if len(in) == 0 {
-		return nil
+		return nil, false
 	}
 
 	if in.AtLeastOneButtonHasDescription() {
 		// We use section layout as we also want to add text description
 		// https://api.slack.com/reference/block-kit/blocks#section
-		return b.renderButtonsWithDescription(in)
+		return b.renderButtonsWithDescription(in), false
 	}
 
 	var btns []slack.BlockElement
 	for _, btn := range in {
 		btns = append(btns, b.renderButton(btn))
 	}
 
+	out := selects.Elements.ElementSet
+	out = append(out, btns...)
 	return []slack.Block{
 		// We use actions layout as we have only buttons that we want to display in a single line.
 		// https://api.slack.com/reference/block-kit/blocks#actions
 		slack.NewActionBlock(
-			"",
-			btns...,
+			selects.BlockID,
```

Comment:
this might be nil and we get a panic, I tried to use kubectl builder and it doesn't work.